python/ray/_private/services.py

```diff
@@ -1645,8 +1678,10 @@ def start_raylet(redis_address,
         f"--max_worker_port={max_worker_port}",
         f"--node_manager_port={node_manager_port}",
         f"--node_ip_address={node_ip_address}",
-        f"--redis_address={gcs_ip_address}",
-        f"--redis_port={gcs_port}",
+        f"--redis_address={redis_ip_address}",
+        f"--redis_port={redis_port}",
+        f"--gcs_address={gcs_ip_address}",
```

Comment:
I thought we decided to use a single flag for gcs ip:port in raylet, or are we going to configure this similar to Redis? If `--gcs_address=` contains only the ip, I will need to update #21115 or fix in a subsequent PR.